cheat/cheatsheets/ssh

```diff
@@ -32,3 +32,8 @@ ssh user@example.com -C -c blowfish -X
 # Copy files and folders through ssh from remote host to pwd with tar.gz compression
 # when there is no rsync command available
 ssh user@example.com "cd /var/www/Shared/; tar zcf - asset1 asset2" | tar zxf -
+
+# Mount folder/filesystem through SSH
+# Install SSHFS from http://fuse.sourceforge.net/sshfs.html
+# Will allow you to mount a folder security over a network.
```

Comment:
s/security/securely/ ?

Also: The SourceForge link just redirects me to https://github.com/libfuse/sshfs…

Also might be worth mentioning that Emacs supports this directly:

    emacs /ssh:name@server/path/to/file

(Also works with `Host`s in SSH config, and `ssh:` can often be inferred)